src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/Payment.cs

```diff
@@ -19,12 +18,18 @@ public Payment(decimal amount, DateTime paymentDate, EarningType earningType)
             PaymentDate = paymentDate;
             EarningType = earningType;
         }
-      
+            
+        public void AddDays(int days)
```

Comment:
This breaks the Immutability of this value object.  Although we have done that elsewhere so need to fix those objects too.